app/src/main/java/org/gdg/frisbee/android/utils/Utils.java

```diff
@@ -218,7 +219,12 @@ public static boolean canLaunch(Context context, final Intent viewUrlIntent) {
 
     @SuppressWarnings("deprecation")
     public static Intent createExternalIntent(Context context, Uri uri) {
-        Intent intent = new Intent(Intent.ACTION_VIEW, uri);
+        CustomTabsIntent customTabsIntent = new CustomTabsIntent.Builder()
+                .setToolbarColor(context.getResources().getColor(R.color.theme_primary))

```

Comment:
You may also use this to access real primaryColor of the theme. I am not really sure which is the best. We are using different colorPrimary for DevFest, StudyJams etc. 

        ThemeUtils.getThemeAttrColor(context, R.attr.colorPrimary);